### DISCUSSION TITLE
hover issue if q-input is embedded in the q-table

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#11966] [General - Components / Directives / etc] [answered]
Dear Quasar team,

I did a demo to illustrate this issue.
https://codepen.io/Stanley-549393092/pen/BawObKz
I have added customized hover (using CSS) to the selected table line.
The issue is when moving mouse to the table line, the table is shaking (not static). However, this not occurs without q-input.
Could you please give some idea, thanks!

**Comment by MilosPaunovic** [comment, marked as answer]
It's a matter of height changing; you're adding a pixel high border on top and on the bottom.

If you style the `td` or your input, this could be avoided:
[https://codepen.io/milospaunovic/pen/poWOYxK](https://codepen.io/milospaunovic/pen/poWOYxK)

**Comment by 549393092** [comment]
@MilosPaunovic Thanks a lot, it works!
